src/current/_includes/v23.2/ui/refresh.md

```diff
@@ -0,0 +1,13 @@
+### Refresh
+
+{% include_cached new-in.html version="v23.2" %} To control refresh of the data on the Active Executions views of the SQL Activity Statements and Transactions pages, the following controls have been added
+
+- an **Active Statements|Transactions As Of** timestamp: Indicates when the last refresh was performed.
+- a manual **Refresh** button: When clicked, refreshes data immediately.
+- an **Auto Refresh** toggle: When toggled **On** (default), refreshes data immediately and then automatically every 10 seconds. When toggled **Off**, stops automatic data refresh. The toggle setting is shared by both the Statements and the Transactions pages. Changing the setting on one page changes it on the other page. 
+
+<img src="{{ 'images/v23.2/active-executions-view-refresh.png' | relative_url }}" alt="Active executions view refresh" style="border:1px solid #eee;max-width:100%" />
+
+If **Auto Refresh** is toggled **On**, navigating to the Active Executions view on either the Statements page or Transactions page, refreshes the data.
```

Comment:
Copy edit: can remove the second comma in this sentence, e.g.

>  **Auto Refresh** is toggled **On**, navigating to the Active Executions view on either the Statements page or Transactions page refreshes the data.

Reply:
updated